using_images/db_images/mysql.adoc

```diff
@@ -62,119 +38,42 @@ This image is available on DockerHub. To download it:
 `$ docker pull openshift/mysql-55-centos7`
 ****
 
-Alternatively, you can build a CentOS 7 based MySQL image:
-
-. Clone the following repository:
-+
-****
-`$ git clone https://github.com/openshift/mysql.git`
-****
-. Navigate into the directory you just created:
-+
-****
-`$ cd mysql`
-****
-. Build the image:
-+
-****
-`$ make build VERSION=5.5`
-****
-
-[NOTE]
-====
-If you omit the `VERSION` parameter, the build action is performed on all
-the provided versions of MySQL. Since version `5.5` is the only verision provided,
-you can omit this parameter.
-====
+To use these images, you can either access them directly in those registries or push them into your OpenShift docker registry.  In addition, it is recommended that you create an ImageStream that points to the image, either in your docker registry or at the external location.  Your OpenShift resources can then reference the ImageStream.  You can find example ImageStream definitions for all the provided OpenShift images https://github.com/openshift/origin/tree/master/examples/image-streams[here].
 
-== Usage
+== Configuration and Usage
 
-To create a container running MySQL, you must run one of following commands,
-depending on your base image.
+=== Initializing the Database
 
-*RHEL 7 Base Image*
+The first time you use the shared volume and initialize the database, the
+database is created along with the database administrator user and the MySQL
+root user (if you specify the `*MYSQL_ROOT_PASSWORD*` environment variable).
+Afterwards, the MySQL daemon starts up. If you are re-attaching the volume to
+another container, then the database user and the administrator user are not
+created, and the MySQL daemon starts.
 
-To create a container running MySQL with a RHEL 7 base image:
+=== Running MySQL Commands in Containers
 
-****
-`$ docker run -d -e MYSQL_USER=_<user>_ -e MYSQL_PASSWORD=_<pass>_ -e MYSQL_DATABASE=_<db>_ -p 3306:3306 openshift/mysql-55-rhel7`
-****
-
-This creates a container named with the credentials and database that you
-specified.
-
-*CentOS 7 Base Image*
-
-To create a container running MySQL with a CentOS 7 base image:
-
-****
-`$ docker run -d -e MYSQL_USER=_<user>_ -e MYSQL_PASSWORD=_<password>_ -e MYSQL_DATABASE=_<db>_ -p 3306:3306 openshift/mysql-55-centos7`
-****
-
-To access and test if database is running:
-
-****
-`$ mysql -h 127.0.0.1 --user=_<user>_ --password=_<password>_ <<< "SELECT 1;"`
-****
+OpenShift uses https://www.softwarecollections.org/[Software Collections] to
+install and launch MySQL. If you want to execute a MySQL command inside of a
+running container (for debugging), you must prefix it with the `scl enable
+mysql55` command.
```

Comment:
I assume the mysql example is supposed to end in "-p password" and not just "-p"